### PR TITLE
docs updating `AzureML` notebooks

### PR DESCRIPTION
**Pull request by leo-gan**
- Added/updated descriptions and links

### CODE REVIEW

File: docs/docs/integrations/chat/azure_chat_openai.ipynb
Comment (efriis) [inline docs/docs/integrations/chat/azure_chat_openai.ipynb:1]: This description might be more confusing than it's worth. The old one is more concise and gets to the point?

Comment (efriis) [reply docs/docs/integrations/chat/azure_chat_openai.ipynb:1]: I think you might have found a different docs page! Let's link this one instead: https://learn.microsoft.com/en-us/azure/ai-services/openai/overview

Comment (leo-gan) [reply docs/docs/integrations/chat/azure_chat_openai.ipynb:1]: We have a separate page for [Azure OpenAI models](azure_chat_openai) . ^ this is about the different integration. It is based on the `AzureMLChatOnlineEndpoint` class. 
So, I think the updated version is correct.

Comment (efriis) [reply docs/docs/integrations/chat/azure_chat_openai.ipynb:1]: I think you've linked the file that I commented on, which I'm pretty sure is about AzureChatOpenAI.

The other file you commented on looks fine. This file is about AzureChatOpenAI though, not the AzureML Endpoints.

Comment (leo-gan) [reply docs/docs/integrations/chat/azure_chat_openai.ipynb:1]: Oops. Yes, you are right. I'm commenting about another file. Sorry. 
Yes, your links is better. Thanks!

**Review by efriis — APPROVED** [review]

**Pull request merged**